webpack/webpack.production.js

```diff
@@ -15,6 +15,7 @@ delete dependencies["canvas-ui"];
 delete dependencies["cnvs"];
 
 module.exports = merge(common, {
+  mode: "production",
```

Comment:
will this replace `"process.env.NODE_ENV"`?

Reply:
Yes; webpack sets this and other behaviors like uglify automatically in this mode

Reply:
I didn't intend for this to be a surprise but left the background in the commit messages . I'll be sure to add these to the PR description in the future https://webpack.js.org/concepts/mode/

Reply:
Oh, my bad. Don't have a strong habit of reading commit messages during a review